forage-android/src/main/java/com/joinforage/forage/android/ForageSDK.kt

```diff
@@ -269,17 +269,21 @@ class ForageSDK : ForageSDKInterface {
     }
 
     /**
-     * Capture a customer's PIN for an EBT payment and defer the capture of the payment to the server
+     * Submits a customer's PIN via a [ForagePINEditText][com.joinforage.forage.android.ui.ForagePINEditText] Element and defers payment capture to the server.
+     *
+     * @param DeferPaymentCaptureParams A model that passes a [`foragePinEditText`][com.joinforage.forage.android.ui.ForagePINEditText] instance and a `paymentRef`, returned by the [Create a Payment](https://docs.joinforage.app/reference/create-a-payment) endpoint, as the [DeferPaymentCaptureParams].
```

Comment:
Not sure if `model` is the most accurate term here

@devinmorgan to comment on this

Reply:
I struggled so hard with this one, @djoksimo and @devinmorgan ! I thought "model" was industry standard because I saw it for similar params on [another docs site](https://stripe.dev/stripe-android/payments-core/com.stripe.android.model/-payment-method-create-params/index.html).

Reply:
Today I learned: Stripe docs call their `*Param` classes "models" 🤔 

@djoksimo open to hearing other names if you have any but I'm OK with model given this info

Reply:
Let's stick with models!